clang/bindings/python/tests/cindex/test_token_kind.py

```diff
@@ -10,26 +10,6 @@
 
 
 class TestTokenKind(unittest.TestCase):
-    def test_constructor(self):
```

Comment:
Can't we keep those tests?

Reply:
thNot really:
- `test_constructor`: this tests the ability to add enum variants "on the fly", which is not possible with the Python stdlib's `Enum` class
- `test_bad_register`: the `Enum` class tests for this on initialization, so if you define a duplicate `Enum` variant, you will get a failure on import already. This makes the test both a) effectively covered by that and b) you couldn't import the module to run such a test anyway in a case where there's a duplicate
- `test_unknown_value`: this is covered in `test_from_id` in `test_enums.py` since I added the `TokenKind` class to the test there

Reply:
> test_constructor: this tests the ability to add enum variants "on the fly", which is not possible with the Python stdlib's Enum class

Do we have a test that this is not possible anymore?

Reply:
So the existing "custom" enum implementation included a sort of constructor, as we see in this test, that takes a number and name and then adds that as a new variant to the existing enum. No such constructor exists for the builtin `Enum` class, so there isn't really an equivalent for this test under the new implementation.
We could test that it is impossible for the user to add their own variants dynamically at runtime by doing something like `TokenKind.NEW_VARIANT = 6`, but that doesn't really seem like our problem, and was also possible under the old implementation

Your comment did however raise another question for me: while we've covered that there are no duplicate names with different IDs, we haven't covered that there are no duplicate IDs with different names, and this is indeed possible in the enum class, and not covered by our current tests so I just added one.